fp.js

```diff
@@ -6,9 +6,10 @@ var people = [
   {name: "Jessica", age: 34},
   {name: "Jasmine", age: 39}
 ]
-people.forEach(function(person){
-  peoplesAges.push(person.age)
-});
+
+people.map(function(person){
```

Comment:
`.map` returns a new array as described by our callback function. In the callback we `return` the modified element to go into that new array:

```js
peoplesAges = people.map(function(person){
    return person.age;
}
```